File: Baekjoon/BaaarkingDog/0x02_basic/2576.py
# 2576: 홀수

min = 100
sum = 0

for _ in range(7):
  num = int(input())

  if (num % 2 == 1):
    sum += num
    if (min > num): min = num

print(-1 if sum == 0 else f"{sum}\n{min}")

File: Baekjoon/BaaarkingDog/0x02_basic/2576.cpp
// 2576: 홀수

#include <iostream>

int main() {
  int min = 100, sum = 0;

  for (int i=0; i<7; i++) {
    int num;
    std::cin >> num;

    if (num % 2) {
      sum += num;
      if (min > num) min = num;
    }
  }

  if (sum) std::cout << sum << std::endl << min;
  else std::cout << -1;

  return 0;
}